cps_data/finalprep.py

```diff
@@ -98,6 +97,24 @@ def main():
     subprocess.check_call(["gzip", "-nf", "cps.csv"])
 
 
+def split_se_income(data):
+    """
+    Split self-employment data into e00900, e02000, and e26270 according to
+    IRS income data
+    """
+    total_se = data.JCPS25 + data.JCPS35
+    earnings_split_p = data.JCPS25 / total_se
+    earnings_split_s = 1. - earnings_split_p
+    data['e00900'] = total_se * 0.318327505
```

Comment:
Sorry to pop in here, but I'm curious if the logic for producing constants like this 0.318 is documented outside the PR? It sounds like it's based on a few cells in the [2014 IRS data](https://www.irs.gov/pub/irs-soi/14in14ar.xls) ([spreadsheet](https://docs.google.com/spreadsheets/d/1LgWl68KveoGfjZbO8Aznpn6M_AsdqDBWs4LzCWVfW2g/edit?usp=sharing)), but I'm not totally clear on exactly which ones.

Reply:
It's based on U9-W9 (business or profession), BA9-BC9 (total rent or royalty), BE9-BG9 (partnership and S corporation), and BI9-BK9 (estate and trust). But I'm looking at alternative methods to get at these variables so I wouldn't put too much time into working this out just yet.